notice_and_comment/static/regulations/css/less/typography-custom.less

```diff
@@ -21,26 +21,31 @@ h6 {
 }
 
 h1 {
-    font-family: 'Source Sans Pro', Georgia, sans-serif;
-    font-weight: 300;
-    font-size: 44px;
-    line-height: 50px;
+  font-weight: 300;
+  font-size: 44px;
+  line-height: 50px;
 }
 
-h2, h3.section-title {
-    font-size: 34px;
-    line-height: 40px;
-    margin: 20px 0;
+h2 {
+  font-weight: 300;
+  font-size: 44px;
+  line-height: 50px;
 }
 
 h3 {
-    font-size: 24px;
-    line-height: 29px;
+  font-size: 34px;
+  line-height: 40px;
+}
+
+h4 {
+  font-size: 24px;
+  line-height: 29px;
+  text-transform: none;
```

Comment:
Should this be applied to `h5` as well? Both are `uppercase`d in -site's `typography.less`